.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_USER=techadmin
+REACT_APP_API_PW=d3vTEAMapi2018!
```

Comment:
Change this please PW and put this file into gitignore

Reply:
done